.projenrc.js

```diff
@@ -115,29 +115,42 @@ for (const lang of ['typescript', 'python', 'java']) {
   project.gitignore.exclude(output);
 }
 
+// backport task to branches based on pr labels
+const backportTask = project.addTask('backport');
+backportTask.exec('npx backport --accesstoken ${GITHUB_TOKEN} --pr ${BACKPORT_PR_NUMBER} --non-interactive');
```

Comment:
It looks like these environment variables might be required. If so we can add a runtime check for this:
```suggestion
// backport task to branches based on pr labels
const backportTask = project.addTask('backport', {
  requiredEnv: ['GITHUB_TOKEN', 'BACKPORT_PR_NUMBER'],
});
backportTask.exec('npx backport --accesstoken ${GITHUB_TOKEN} --pr ${BACKPORT_PR_NUMBER} --non-interactive');
```

Reply:
Ohh nice, didn't know about that - thanks!